examples/predictors/lstm/Example-LSTM-Predictor.ipynb

```diff
@@ -404,7 +406,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!python predict.py -s 2020-08-01 -e 2020-08-04 -ip ../../../validation/data/2020-09-30_historical_ip.csv -o predictions/2020-08-01_2020-08-04.csv"
+    "!head predictions/2020-08-01_2020-08-04.csv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test cases\n",
```

Comment:
Same cases as for the template and for the Linear example. They don't pass yet: that will be for #92